src/components/wishlist/WishListInfo.jsx

```diff
@@ -0,0 +1,54 @@
+import styled from 'styled-components';
+
+function WishListInfo(categoryList) {
+  const list = categoryList.list;
+
+  const getCategoryInfo = () => {
+    return list.map(({ id, image, title }) => (
+      <li key={id}>
+        <img src={image}></img>
+        <span>{title}</span>
+      </li>
+    ));
+  };
+
+  return (
+    <StyledWishListInfo>
+      <StyledCategoryInfo>{getCategoryInfo()}</StyledCategoryInfo>
+    </StyledWishListInfo>
+  );
+}
+
+export default WishListInfo;
+
+const StyledWishListInfo = styled.div`
+  border-style: dashed;
+  padding: 10.4rem 2.2rem;
```

Comment:
```suggestion
  padding: 0 2.2rem;
```
위에 Header 수정하면 이렇게 바꿔주셔도 됩니당!